hunts/src/puzzlesSlice.js

```diff
@@ -55,6 +63,12 @@ export const puzzlesSlice = createSlice({
     [fetchPuzzles.fulfilled]: (state, action) => {
       puzzlesAdapter.setAll(state, action.payload);
     },
+    [updatePuzzle.fulfilled]: (state, action) => {
+      puzzlesAdapter.updateOne(state, {
+        id: action.payload.id,
+        changes: { ...action.payload },
+      });
+    },
```

Comment:
Is there any indication for .rejected? We can add it later, though it'd be helpful to add a TODO as a reminder.

Reply:
Error handling is done elsewhere: https://github.com/cardinalitypuzzles/smallboard/blob/master/hunts/src/alertSlice.js#L29-L34 for displaying error alerts (similar to the non-react UI)